File: android/day02Test/app/src/main/java/com/example/day02test/SecondActivity.java
package com.example.day02test;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        TextView textView2 = findViewById(R.id.textView2);

        Intent i = getIntent();
        String data = i.getStringExtra("key");

        textView2.setText(data);
    }
}

File: android/study/Hilt_1/app/src/main/java/com/example/hilt_1/ui/second/SecondActivity.kt
package com.example.hilt_1.ui.second

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.hilt_1.R
import com.example.hilt_1.data.MyRepository
import dagger.hilt.android.AndroidEntryPoint
import javax.inject.Inject

@AndroidEntryPoint
class SecondActivity : AppCompatActivity() {

    @Inject
    lateinit var repository: MyRepository

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
    }
}